src/h5web/dataset-visualizer/vis/MatrixVis.module.css

```diff
@@ -13,3 +13,33 @@
 .cell[data-bg] {
   background-color: var(--secondary-light-bg);
 }
+
+.indexCell {
+  display: flex;
+  font-weight: bold;
+  align-items: center;
+  justify-content: center;
+  background-color: var(--primary-bg);
+}
+
+.indexContainer {
+  display: flex;
+  z-index: 2;
```

Comment:
Would `1` work?

Reply:
Actually... When you scroll horizontally, it looks like `indexRow` goes above `indexColumn`, but when you scroll vertically it's actually `indexColumn` that goes above `indexRow`. This inconsistent behaviour looks a little strange.

It comes from the fact that:
- the empty index cell in the top left of the grid belongs to `indexRow`,
- `indexColumn` comes after `indexRow` in the DOM, so its the stacking order is higher than `indexRow`'s.

I think the simpler option to fix this would be for `indexRow` to have `z-index: 2` and `indexColumn` to have `z-index: 1`. This means that the row will always remain above the column. If the opposite behaviour is preferred, then an empty cell would have to be added to `indexColumn` in order to cover the row when scrolling horizontally (the column would also have to be negatively offset).

Reply:
Looks good :+1:

Reply:
Yes, much better!

I actually had an idea during the demo: what if the anchor cell always stayed in the corner when `sticky` is enabled? Both index bands would then scoll under it. I think it may look even better!

You could render the anchor cell inside the `InnerGrid` component directly instead of inside the index row. (The index row would then have to be aligned to the right with `justify-content: flex-end` to account for the missing cell). The anchor cell would be positioned with `position: sticky`, `top: 0` and `bottom: 0` relative to the inner grid `div` (although `position: fixed` would work as well).

What do you reckon?

Reply:
I thought about it as well but was unsure on how to do it. Your proposal looks sensible, I will try to implement that !